pkgs/development/python-modules/aider-chat/default.nix

```diff
@@ -1,15 +1,15 @@
 {
   lib,
   stdenv,
-  python312,
+  python,
   fetchFromGitHub,
   gitMinimal,
   portaudio,
   playwright-driver,
 }:
 
 let
-  python3 = python312.override {
+  python3 = python.override {
```

Comment:
The module for aider-chat is incorrect, but the application needs 3.12 to work. I know i need to fix this, but for a proper fix, the python version needs to be set in the package.nix 
Aider doesnt work with 3.13 yet

Reply:
I'll set python312Packages there, then